api.bs

```diff
@@ -631,6 +641,36 @@ as long as <a method for=PrivateAttribution>measureConversion()</a>
 excludes expired [=impressions=] from [=attribution=]. However, the
 [=user agent=] should not retain expired [=impressions=] indefinitely.
 
+
+### Clearing ### {#impression-store-clearing}
+
+A mechanism must be provided to clear the impression store.
+For example, the impression store could be cleared
+upon activation of the control that
+[[#opt-out|disables]] the Private Attribution API.
+It is recommended that any mechanism a user agent provides
+to clear stored browsing data (history, cookies, etc.)
+be extended to cover the impression store.
```

Comment:
A possible open issue:

While clearing impressions when the browser clears *all* state is reasonable, what would we do for clearing the state for individual sites?  Right now, if we attribute the stored impressions to the conversion site, we'll be in a pretty good state. I think.   If that site clears state, then it loses impressions.  Does that come with risks?

Reply:
We should also make it obvious that clearing state from the browser DOES NOT clear privacy budget state.  It might consolidate it though (for instance, it might say that all budgets for weeks prior to the clearing is consumed entirely).